### PR TITLE
SQ Plugin: Bump orchestrator to 5.0.0.2065

### LINKED ISSUE #9549: SQ Plugin: Bump orchestrator to 5.0.0.2065
See #9476, v5 breaks assertions on some ITs, because timestamps are added.

ToDo:
* Check with AE squad if this is expected
* Adjust our assertions if needed

### PR DESCRIPTION
**Pull request by mary-georgiou-sonarsource**
Fixes #9549

### CODE REVIEW

File: its/src/test/java/com/sonar/it/shared/ScannerCliTest.java [hunk 1]
```diff
@@ -65,10 +66,11 @@ void givenMainHtmlCodeAndTestCSharpCode_whenScannerForCliIsUsed_logsCSharpWarnin
     BuildResult result = ORCHESTRATOR.executeBuild(scanner);
 
     assertThat(result.getLogsLines(l -> l.contains("WARN")))
-      .containsExactlyInAnyOrder(
-        "WARN: Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",
-        INCREMENTAL_PR_ANALYSIS_WARNING
-      );
+      .hasSize(2)
+      .doesNotHaveDuplicates()
```
Comment (zsolt-kolbay-sonarsource) [inline its/src/test/java/com/sonar/it/shared/ScannerCliTest.java:70]: Not needed. You check for the size of 2, then check for two different strings.

**Review by zsolt-kolbay-sonarsource — COMMENTED** [review]
Looks good, but the assertions need some changes.

File: its/src/test/java/com/sonar/it/shared/ScannerCliTest.java [hunk 2]
```diff
@@ -46,12 +46,13 @@ void givenRazorPagesMainCode_whenScannerForCliIsUsed_logsCSharpWarning() {
...
 
     assertThat(result.getLogsLines(l -> l.contains("WARN")))
-      .containsExactlyInAnyOrder(
-        "WARN: Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",
-        "WARN: Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",
-        INCREMENTAL_PR_ANALYSIS_WARNING,
-        INCREMENTAL_PR_ANALYSIS_WARNING
-      );
+      .hasSize(4)
+      .doesNotHaveDuplicates()
```
[the remark this answers is not in the document]
Comment (mary-georgiou-sonarsource) [reply its/src/test/java/com/sonar/it/shared/ScannerCliTest.java:50]: True - checking the size to 4 should be enough.

File: its/src/test/java/com/sonar/it/shared/ScannerCliTest.java [hunk 3]
```diff
@@ -46,12 +46,13 @@ void givenRazorPagesMainCode_whenScannerForCliIsUsed_logsCSharpWarning() {
...
+      .anyMatch(x -> x.endsWith("WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
+      .anyMatch(y -> y.endsWith("WARN  Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
+      .anyMatch(w -> w.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING))
+      .anyMatch(z -> z.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING));
```
[the remark this answers is not in the document]
Comment (mary-georgiou-sonarsource) [reply its/src/test/java/com/sonar/it/shared/ScannerCliTest.java:54]: Yes - the intention is to check that the message appears twice.

[the remark this answers is not in the document]
Comment (zsolt-kolbay-sonarsource) [reply its/src/test/java/com/sonar/it/shared/ScannerCliTest.java:53]: But instead, it checks twice whether the collection contains the same element at least once, right?
If you want to check if it's contained exactly twice, then you need to filter the collection for that message and then count the elements.

**Review by zsolt-kolbay-sonarsource — APPROVED** [review]
LGTM. Please check [this comment](https://github.com/SonarSource/sonar-dotnet/pull/9551#discussion_r1691021449) before merging.

**Pull request merged**